homeassistant/helpers/selector.py

```diff
@@ -741,10 +741,11 @@ class TargetSelectorConfig(TypedDict, total=False):
     device: SingleDeviceSelectorConfig
 
 
-class StateSelectorConfig(TypedDict):
+class StateSelectorConfig(TypedDict, total=False):
     """Class to represent an state selector config."""
 
     entity_id: str
+    attribute: str
```

Comment:
We already have an `AttributeSelector`. Wouldn't this be a duplicate?

https://github.com/home-assistant/core/blob/8bd76cb42b5a8a43fa51e3d5062035043dc0061a/homeassistant/helpers/selector.py#L216